main.go

```diff
@@ -65,8 +67,18 @@ func makeStmtMessage(fset *token.FileSet, funcInfo *ast.Ident, stmts, stmtLimit
 	}
 }
 
-func getLines(fset *token.FileSet, f *ast.FuncDecl) int { // nolint: interfacer
-	return fset.Position(f.End()).Line - fset.Position(f.Pos()).Line - 1
+func getLines(fset *token.FileSet, f *ast.FuncDecl, cmap ast.CommentMap) int { // nolint: interfacer
+	var commentCount int
+
+	for _, c := range cmap.Comments() {
+		// If the comment line is inside the function
+		// count how many comments are in the CommentGroup
+		if fset.Position(c.Pos()).Line > fset.Position(f.Pos()).Line {
```

Comment:
I guess I should check the .End.Line as well, to make sure it is really inside the function

Reply:
Updated the condition :heavy_check_mark: